packages/core/admin/_internal/cli/index.ts

```diff
@@ -8,6 +8,7 @@ import type { DevelopCLIOptions } from './commands/develop';
 const build: StrapiCommand = ({ command, ctx }) => {
   command
     .command('build')
+    .option('--bundler [bundler]', 'Bundler to use (webpack or rspack)', 'webpack')
```

Comment:
I just want to think about the drawbacks. This seems to be quite a bit more code to maintain. Do you feel like it's worth it? What scale of performance improvements are we talking about?

Or would the plan be to eventually ditch webpack?

Reply:
>  What scale of performance improvements are we talking about?

Did you check out the branch? It saw some _very_ impressive improvements.
We're not removing webpack in V5 because it just creates an unnecessary barrier for users to overcome but it will be in v6.